docs/Guides/IAP-Integration.md

```diff
@@ -0,0 +1,172 @@
+# IAP Integration
+
+Cloud Pay supports In App Purchases from the following providers:
+
+* Amazon
+* Android
+* Apple
+* Roku
+
+## Setup
+
+### Generic
+
+In order to support IAP a corresponding package must be created in Cloud Pay that matches the IAP package in terms of cost, type (recurring or non recurring) and contract length.  The product ID/s obtained from the IAP provider is used to link the Cloud Pay packages with the IA and is entered via the admin console on Cloud Pay.  This allows us to sync that various states from the providers such as Renewals, Cancellations, Refunds etc and therefore provide access to premium content.
```

Comment:
"The product ID/s obtained from the IAP provider is used to link the Cloud Pay packages with the IA and is entered via the admin console on Cloud Pay."

**Suggested Edit:**
"The product ID/s obtained from the IAP provider is used to link the Cloud Pay packages, this can be achieved by manually entering the IAP product ID within the CloudPay Admin console for the selected package."

"This allows us to sync that various states"

**Suggested Edit:**

"This allows us to sync the various states"